2-0-data-structures-and-algorithms/src/test/java/com/bobocode/stack/StackTest.java

```diff
@@ -0,0 +1,75 @@
+package com.bobocode.stack;
+
+import com.bobocode.stack.exception.EmptyStackException;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.*;
+
+class StackTest {
+
+    private Stack<Integer> intStack = new NodeStack<>();
+
+    @Test
+    void testPushElementOntoEmptyStack() {
+        intStack.push(234);
+    }
+
+    @Test
+    void testPopElementFromEmptyStack() {
+        assertThrows(EmptyStackException.class, () -> intStack.pop());
+    }
+
+    @Test
+    void testPushElements() {
+        intStack = NodeStack.of(23, 35, 72);
+
+        intStack.push(55);
+
+        assertThat(intStack.pop(), is(55));
```

Comment:
@mathodcoast let's use AssertJ and refactor this line to 
`assertThat(intStack.pop()).isEqualTo(55);